src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing/InputCustomImportOrderValid.java

```diff
@@ -1,5 +1,6 @@
 package com.google.checkstyle.test.chapter3filestructure.rule333orderingandspacing;
 
+
```

Comment:
it test updated to prevent future regression.

Reply:
Please add a comment line there to boost certenty that it is on purpose: `// it is not fogbidden to have extra lines ( more then one) between package and import group`

It will be harder to remove it, if somebody try.